src/Uno.UI.Runtime.Skia.Gtk/UI/Xaml/Controls/TextBoxViewExtension.cs

```diff
@@ -425,14 +428,14 @@ public int GetSelectionLength()
 			}
 		}
 
-		public void SetForeground(Windows.UI.Xaml.Media.Brush brush)
+		public void SetForeground(Brush brush)
 		{
-			if (_currentInputWidget is not null && brush is SolidColorBrush scb)
+			if (_currentInputWidget is { } widget && brush is SolidColorBrush scb)
```

Comment:
They are equivalent, but I think `is not null` is more readable

```suggestion
			if (_currentInputWidget is not null && brush is SolidColorBrush scb)
```